OpenDreamRuntime/DreamValue.cs

```diff
@@ -74,7 +75,7 @@ public enum DreamValueType {
                 DreamObject => DreamValueType.DreamObject,
                 DreamPath => DreamValueType.DreamPath,
                 DreamProc => DreamValueType.DreamProc,
-                _ => throw new ArgumentException("Invalid DreamValue value (" + value + ", " + value.GetType() + ")")
+                _ => DreamValueType.Reference
```

Comment:
Could this be changed to accept only `IDreamProcIdentifier` and `DreamProcArguments` to keep the error handling for invalid values?

Reply:
that's a good idea